skalenetwork/skaled
# FTQ rpc call, v3.18.0 again

Note: in results, transactions belonging to same sender go together, ordered by nonce increase. In case of FTQ overflow, last transaction from 1st sender will be dropped.

Testing:

1 JsonRpcSuite/mtm_import_future_txs

2 Manual testing sequence:
  1. Add balance for your account to skaled's config.
  3. Run skaled with --enable-debug-behavior-apis
  4. Send transaction with too big nonce.
  5. Send curl -X POST --data '{"id":4,"jsonrpc":"2.0","method":"debug_getFutureTransactions","params":[]}'
  6. See your transaction.
  7. Repeat the same without --enable-debug-behavior-apis, METHOD_NOT_FOUND error should be returned.

Example answer:
```
{
   "id":4,
   "jsonrpc":"2.0",
   "result":[
      {
         "from":"0xf15f970e370486d5137461c5936dc6019898e6c8",
         "gas":"0x5208",
         "gasPrice":"0x186a0",
         "hash":"0xf075c0d864e4eb73218480b83ff231a74846acaf58136f310f2241fd4e8beb8b",
         "nonce":"0x14",
         "r":"0x5d18118a4f1d559af9bcd6728066da2b1786ae6acd9d3785a93b2561706f2f48",
         "s":"0x29e5a3fef45fd3d606aeac4e602eeb7389d50af6f708179b226eed5252e702d9",
         "sighash":"0xa075eb68e97bd62f14315ccd9faab3d0c3119db609a997e568c52387042ffae7",
         "to":"0xf15f970e370486d5137461c5936dc6019898e6c8",
         "v":"0x1",
         "value":"0x0"
      }
   ]
}
```

Example error:
```{"error":{"code":-32601,"message":"METHOD_NOT_FOUND: The method being requested is not available on this server"},"id":4,"jsonrpc":"2.0"}```
